pkgpanda/actions.py

```diff
@@ -141,14 +135,14 @@ def remove_package(install, repository, package_id):
         PackageId(package_id)
         repository.remove(package_id)
     except ValidationError:
-        print("\nInvalid package id {0}".format(package_id))
-        sys.exit(1)
+        raise ValidationError("Invalid package id {0}".format(package_id))
```

Comment:
Should be a `raise ValidationError() from ex` so we keep the exception history (same goes with the other re-throw code which rewrites the exception being thrown)